wallet/build.gradle

```diff
@@ -25,6 +25,7 @@ dependencies {
     implementation 'com.google.protobuf:protobuf-java:3.7.1'
     implementation 'com.google.guava:guava:29.0-android'
     implementation 'com.google.zxing:core:3.3.3'
+    implementation 'androidx.appcompat:appcompat:1.3.1'
```

Comment:
The app deliberately isn't using AppCompat. I think you can archieve the same thing using the standard views.

Reply:
OK, I will try to. The missing space is easy to handle. But could you tell me the reason why you deliberately do not use AppCompat? Is it because of some latent compatibility issues? Thanks.